src/types/view/index.ts

```diff
@@ -44,6 +44,12 @@ export interface ViewSubmitAction {
   view: ViewOutput;
   api_app_id: string;
   token: string;
+  // exists for enterprise installs
+  is_enterprise_install?: boolean;
```

Comment:
Haven't confirmed if the payload is actually getting these two keys yet in enterprise install (and that team is null)